discord.html

```diff
@@ -1,200 +1,186 @@
-                        progress-updates:All info related to current ongoing
-                        task,discussions are posted here.
-                      </li>
-                      <li class="channel-name">
-                        support:In case you are stuck on need some help everyone
-                        is there to help you keep running.
-                      </li>
-                    </ol>
-                  </li>
+                <li class="">
+                  <strong>Main: </strong> It has channels for general
+                  disussion,progress-updates ,newcomers & Support.
+                  <ol class="channel-info">
+                    <li class="channel-name">
+                      <b>#general:</b> It carries general discussions.
+                    </li>
+                    <li class="channel-name">
+                      <b>#newcomers:</b> All info related to newcomers are floated in
+                      it.
+                    </li>
+                    <li class="channel-name">
+                      <b>#progress-updates:</b> All info related to current ongoing
+                      task,discussions are posted here.
+                    </li>
+                    <li class="channel-name">
+                      <b>#support:</b> In case you are stuck on need some help everyone
+                      is there to help you keep running.
+                    </li>
+                  </ol>
+                </li>
 
-                  <li class="category-name">
-                    <strong>Text:</strong> Contains multiple other channels in
-                    it.
-                  </li>
+                <li class="category-name">
+                  <strong>Text:</strong> Contains multiple other channels in
+                  it.
+                </li>
 
-                  <li class="category-name">
-                    <strong>Learning:</strong> It contains different technology
-                    related channels plus channel for interview preparation.
-                  </li>
+                <li class="category-name">
+                  <strong>Learning:</strong> It contains different technology
+                  related channels plus channel for interview preparation.
+                </li>
 
-                  <li class="category-name">
-                    <strong>Dev groups:</strong> It contains channels for
-                    different projects.
-                  </li>
+                <li class="category-name">
+                  <strong>Dev groups:</strong> It contains channels for
+                  different projects.
+                </li>
 
-                  <li class="category-name">
-                    <strong>Tasks:</strong> Channels for seperate tasks.
-                  </li>
-                </ul>
-              </div>
+                <li class="category-name">
+                  <strong>Text Channels : </strong> Channels for seperate tasks.
```

Comment:
https://github.com/Real-Dev-Squad/website-welcome/pull/150#discussion_r601620963

Here it is `Tasks`!

Reply:
done